bpf/kmesh/ads/include/cluster.h

```diff
@@ -20,6 +22,22 @@ struct {
     __uint(max_entries, MAP_SIZE_OF_CLUSTER);
 } map_of_cluster SEC(".maps");
 
+struct inner_of_maglev {
+    __uint(type, BPF_MAP_TYPE_ARRAY);
+    __uint(max_entries, 1);
+    __uint(key_size, sizeof(__u32));
+    __uint(value_size, sizeof(__u32) * MAGLEV_TABLE_SIZE);
+};
+
+struct {
+    __uint(type, BPF_MAP_TYPE_HASH_OF_MAPS);
+    __uint(key_size, CLUSTER_NAME_MAX_LEN);
+    __uint(value_size, sizeof(__u32));
+    __uint(max_entries, MAP_SIZE_OF_CLUSTER);
+    __uint(map_flags, BPF_F_NO_PREALLOC);
+    __array(values, struct inner_of_maglev);
+} outer_of_maglev SEC(".maps");
+
```

Comment:
Is it necessary to use the outer-inner map structure here?

Reply:
Will be done in next pr.